### PR TITLE
[i18n] Chinese Translation and add bilibili video integration

### PR DESCRIPTION
**Pull request by btspoony**
Following Content Translations (Text) #2 guidelines

We have produced Chinese translations and added a review video for each course which comes from our technical ambassadors in the Chinese area.
Considering that China Mainland can not access to YouTube, we have added BiliBili video integration.

### CODE REVIEW

**Comment by kerrywei** [comment]
Thank you so much for providing the Chinese translation! I am happy to help take a look later this week :)

File: src/missions/FAQ.cn.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+### 谁是 CryptoDappy 背后的那个人?
+
+CryptoDappy 是 Dapper Labs 技术市场经理 Benjamin Ebner 的个人项目，
+但它不是 Flow 的官方项目。
+
+### 在哪里我能得到帮助呢?
+
+查看我们的办公时间以寻求帮助，或加入我们的 Discord，
```
Comment (kerrywei) [inline src/missions/FAQ.cn.mdx]: we should consider linking the 'Discord' word to our Chinese channel?

```suggestion
查看我们的办公时间以寻求帮助，或加入我们的 [Discord](https://discord.com/channels/613813861610684416/751491594989076502)，
```

File: src/missions/contribute.cn.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+我们正在积极寻找想要贡献自己和/或
+想要拥有 CryptoDappy 及其 learning hub 某些地方所有权的开发人员，尤其是在以下领域：
+
+* Learning hub 的国际化(文本，音频)
+* 进一步完善 CryptoDappy 的额外功能
+* 改进当前应用程序的设计
+
+在开始改善我们之前，请阅读我们的 
+[改善指南](https://github.com/bebner/crypto-dappy/blob/master/CONTRIBUTING.md) 
+并前往我们的 main repositories 提出 issues 或提出 pull request：
```
Comment (kerrywei) [inline src/missions/contribute.cn.mdx]:
```suggestion
并前往我们的代码仓库提出反馈或提交合并请求：
```

File: src/missions/contribute.cn.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+* [CryptoDappy](https://github.com/bebner/crypto-dappy)
+* [CryptoDappy Learning Hub](https://github.com/bebner/crypto-dappy-learning-hub)
+
+在 "issues"选项卡下，你已经找到了各种 good first issues, 它们对任何想深入了解加密货币的人来说都是不错的选择。
```
Comment (kerrywei) [inline src/missions/contribute.cn.mdx]:
```suggestion
在 "issues"选项卡下，你可以找到适合新手的各种 [good first issues](https://github.com/bebner/crypto-dappy-learning-hub/labels/good%20first%20issue), 它们对任何想深入了解加密货币的人来说都是不错的选择。
```

File: src/missions/mission-1.cn.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+<img src="https://d1mtab9ltfdpua.cloudfront.net/images/Dappy11.png" width="20%" style="float: left;" />
+
+
+### Peter Dappy
+
+产品经理
+
+*嘿! 非常棒的入门介绍*👍
```
Comment (kerrywei) [inline src/missions/mission-1.cn.mdx]:
```suggestion
*嘿! 恭喜你完成入门任务*👍
```

File: src/missions/mission-1.cn.mdx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+*这是为了让用户能够在未来进行 Dappy 的收集和支付
+
+我们能找到一个区块链的解决方案吗? 感恩🙏*
+
+---
+
+## 任务概述
+
+This mission is all about authentication with the blockchain: Signing users in and out of their application by using a wallet.
```
Comment (kerrywei) [inline src/missions/mission-1.cn.mdx]:
```suggestion
```

File: src/missions/mission-2.cn.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+# 任务 #2
+
+大事不好了，就在首月进程之中，我们遇到了一个大问题，达福（Dappy）数据库被人侵入了！截至目前，我们还尚未让达福的身影现身于区块链之中，这也让其十分容易被篡改—这并非是获取新用户的好方法，而且他们也无法真正拥有达福。正因如此，达福目前已经被转移到其区块链上的智能合约之中。Tarek想请你帮个忙，快来看看怎么回事吧。.
+
+------
+
+[![46msyQ.png](https://z3.ax1x.com/2021/09/26/46msyQ.png)](https://imgtu.com/i/46msyQ)
+
+### Tarek Dappy
```
Comment (kerrywei) [inline src/missions/mission-2.cn.mdx:22]:
```suggestion
### Tarek Dappy (职位：初级开发员)

```

File: src/missions/mission-2.cn.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+初级开发员
```
Comment (kerrywei) [inline src/missions/mission-2.cn.mdx:24]:
```suggestion
```

File: src/missions/mission-1.cn.mdx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+您仍在解决所有问题并与团队中的每个人会面时，天才产品经理 Peter Dappy 突然在 Slack 上向您发送信号。 
+看起来他忘记了这个应用程序中的一个重要部分，您将不得不处理这件事：身份验证。
+
+---
+
+<img src="https://d1mtab9ltfdpua.cloudfront.net/images/Dappy11.png" width="20%" style="float: left;" />
+
+
+### Peter Dappy
```
Comment (kerrywei) [inline src/missions/mission-1.cn.mdx:18]:
```suggestion
### Peter Dappy (职位：产品经理)
```

File: src/missions/mission-1.cn.mdx [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+产品经理
```
Comment (kerrywei) [inline src/missions/mission-1.cn.mdx:20]:
```suggestion
```

File: src/missions/mission-3.cn.mdx [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+是的，你的游戏越来越受欢迎，你的用户喜欢可爱的小 Dappies！
+您与产品团队一起决定让用户在他们的个人收藏中收集 Dappies。
+但要实现这一点，您必须通过与智能合约进行交互来实现交易。
+
+---
+
+<img src="https://d1mtab9ltfdpua.cloudfront.net/images/Dappy4.png" width="20%" style="float: left;" />
+
+### Liza Dappy
```
Comment (kerrywei) [inline src/missions/mission-3.cn.mdx:18]:
```suggestion
### Liza Dappy （职位：高级开发人员）
```

File: src/missions/mission-3.cn.mdx [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+高级开发人员
```
Comment (kerrywei) [inline src/missions/mission-3.cn.mdx:20]:
```suggestion
```

File: src/missions/mission-4.cn.mdx [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+import YoutubeEmbed from "../components/youtube.js"
+import BilibiliEmbed from "../components/bilibili.js"
+
+随着你对区块链的了解日渐加深，你的同事也渐渐发觉了你掌握的新技能！厉害！因此用不了多久，你的同事就会在Slack上问你相关问题，这次带来了一个重要的任务：让用户们使用FUSD作为Dappy的支付方式。
+---
+
+<img src="https://d1mtab9ltfdpua.cloudfront.net/images/Dappy12.png" width="20%" style="float: left;" />
+
+### Hana Dappy
```
Comment (kerrywei) [inline src/missions/mission-4.cn.mdx:15]:
```suggestion
### Hana Dappy（职位：资深开发人员）
```

File: src/missions/mission-4.cn.mdx [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+资深开发人员
```
Comment (kerrywei) [inline src/missions/mission-4.cn.mdx:17]:
```suggestion
```

File: src/missions/mission-4.cn.mdx [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+## 同质化代币：概览
+
+FUSD是Flow上遵循同质化代币标准的一种代币，因此有必要了解同质化代币的概念。
+
+<YoutubeEmbed embedId="knfQSfkxBi0"/>
+
+## FUSD：查询Vault & 余额
+
+After the high-level concepts out of the way, we'll now be jumping into the code to implement FUSD, 
+checking for the vault of a user and the specific FUSD balance.
```
Comment (kerrywei) [inline src/missions/mission-4.cn.mdx:51]: we can remove the English text now?

File: src/missions/mission-5.cn.mdx [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+第一个关键时刻即将到来：Dappy铸币! 
+你和团队正在为这个重要的里程碑努力工作，因为最后期限正在慢慢接近。
+你准备好迎接下一个版本了吗？
+
+
+---
+
+<img src="https://d1mtab9ltfdpua.cloudfront.net/images/Dappy10.png" width="20%" style="float: left;" />
+
+### Chris Dappy
```
Comment (kerrywei) [inline src/missions/mission-5.cn.mdx:18]:
```suggestion
### Chris Dappy（职位：解决方案工程师）
```

File: src/missions/mission-5.cn.mdx [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+
+解决方案工程师
```
Comment (kerrywei) [inline src/missions/mission-5.cn.mdx]:
```suggestion
```

File: src/missions/mission-6.cn.mdx [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+
+# 任务 #6
+
+你做到了，达福（Dappy）铸币已经运作，而且你的用户都爱死它了。你现在每天都会迎来几千个新注册的用户。但对部分人来说，单一达福铸币远远不够：他们想要的是Dappy Packs! 
+
+---
+
+[![46n4AI.png](https://z3.ax1x.com/2021/09/26/46n4AI.png)](https://imgtu.com/i/46n4AI)
+
+### Lara Dappy
```
Comment (kerrywei) [inline src/missions/mission-6.cn.mdx]:
```suggestion
### Lara Dappy（职位：产品经理）

```

File: src/missions/mission-6.cn.mdx [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+
+产品经理
```
Comment (kerrywei) [inline src/missions/mission-6.cn.mdx]:
```suggestion
```

File: src/missions/onboarding.cn.mdx [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
+
+团队已经准备好了专用的学习任务，让你能够更快的掌握相关技术知识，并且同时为他们V1.0的收藏品游戏CryptoDappy做贡献！
+
+在了解完概要之后，现在是时候深入了解这个应用最初结构和所用格式的细节了。如果你对React Hooks不是很了解的话，请先看一下这两个链接[React hooks](https://reactjs.org/docs/hooks-intro.html) 和 [Context API](https://reactjs.org/docs/context.html)
+
+---
+
+<img src="https://d1mtab9ltfdpua.cloudfront.net/images/Dappy1.png" width="20%" style="float: left;" />
+
+### Dana Dappy
```
Comment (kerrywei) [inline src/missions/onboarding.cn.mdx]:
```suggestion
### Dana Dappy（职位：HR经理）
```

File: src/missions/onboarding.cn.mdx [hunk 20]
```diff
@@ -0,0 +1,91 @@
...
+
+HR 经理
```
Comment (kerrywei) [inline src/missions/onboarding.cn.mdx]:
```suggestion
```

**Review by kerrywei — COMMENTED** [review]
super great to see this translation work (and especially the videos)! 
we should consider adding UI changes to allow users to pick language of the content? right now I can't switch locale in the learning hub. 
no other comments on the translation. good job!

**Comment by btspoony** [comment]
> super great to see this translation work (and especially the videos)! we should consider adding UI changes to allow users to pick language of the content? right now I can't switch locale in the learning hub. no other comments on the translation. good job!

Thanks for the review and all these suggested changes!

And here is a solution for switching locale Done in FLIP Fest.
Issue: <https://github.com/onflow/flip-fest/issues/6>
Merged PR: <https://github.com/onflow/flip-fest/pull/51>

**Review by kerrywei — APPROVED** [review]
The mandarin contents look good to me. Also verified the Bilibili video cant be loaded, which is great
@bebner do you have any comments?

File: src/missions/onboarding.cn.mdx [hunk 21]
```diff
@@ -0,0 +1,90 @@
+---
+title: "入门指南"
+date: "2021-09-27"
+path: '/onboarding'
```
Comment (knagato) [inline src/missions/onboarding.cn.mdx]:
```suggestion
path: '/missions/mission-0'
```

**Review by knagato — COMMENTED** [review]
Hi
I see the path of [onboarding.mdx](https://github.com/bebner/crypto-dappy-learning-hub/blob/master/src/missions/onboarding.mdx) is `/missions/mission-0`

**Comment by btspoony** [comment]
> Hi I see the path of [onboarding.mdx](https://github.com/bebner/crypto-dappy-learning-hub/blob/master/src/missions/onboarding.mdx) is `/missions/mission-0`

Ty

**Pull request merged**